src/client/v2/algod/accountInformation.ts

```diff
@@ -15,4 +15,23 @@ export default class AccountInformation extends JSONRequest {
   path() {
     return `/v2/accounts/${this.account}`;
   }
+
+  /**
+   * Exclude assets and application data from results
+   *
+   * #### Example
+   * ```typescript
+   * const address = "XBYLS2E6YI6XXL5BWCAMOA4GTWHXWENZMX5UHXMRNWWUQ7BXCY5WC5TEPA";
+   * const accountInfo = await algodClient.accountInformation(address)
+   *        .excludeInfo('all')
+   *        .do();
+   * ```
+   *
+   * @param round
+   * @category query
+   */
+  excludeInfo(exclude: string) {
```

Comment:
Should this be an array, or maybe mention in the documentation that something like `created-asset,created-apps` is a valid input?